test/python/utest_masarclient/test_timeStamp.py

```diff
@@ -0,0 +1,120 @@
+import unittest
```

Comment:
As an aside, the TimeStamp class needs to be removed, or dramatically simplified.  In the context of MASAR there is no benefit in having this complexity to keep a level of precision that the backend(s) can't store.